src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -1911,6 +2041,63 @@ public Visitable processMethodDeclaration(MethodDeclaration node) {
     return result;
   }
 
+  /**
```

Comment:
This method should be static, and should be in `JavaLangUtils`, not `UnsolvedSymbolVisitor` (which has too much functionality as it is).